pkg/http/roundtripper.go

```diff
@@ -37,6 +37,18 @@ func NewDebugRoundTripper(logger log.Logger, next http.RoundTripper) *debugRound
 	return &debugRoundTripper{next, log.With(logger, "component", "http/debugroundtripper")}
 }
 
+type TLSRoundTripper struct {
+	wrapper http.RoundTripper
+}
+
+func NewTLSRoundTripper(next http.RoundTripper) *TLSRoundTripper {
+	return &TLSRoundTripper{next}
+}
+
+func (rt *TLSRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	return rt.wrapper.RoundTrip(req)
+}
+
```

Comment:
Is this really needed?

Reply:
There is NewBearerRoundTripper and DebugRoundTripper, it is needed a new function to create a different one, right?

Reply:
done